tests/foreman/ui/test_contentview.py

```diff
@@ -1370,3 +1370,170 @@ def test_positive_delete_version_with_ak(self):
             session.nav.go_to_content_views()
             self.content_views.delete_version(cv.name, version)
             self.content_views.validate_version_deleted(cv.name, version)
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier2
+    def test_positive_add_custom_ostree(self):
+        """Create a CV with custom ostree contents
+
+        @Feature: Content View
+
+        @Assert: CV should be created successfully with custom ostree contents
+
+        @Status: Manual
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier2
+    def test_positive_add_rh_ostree(self):
+        """Create a CV with RH ostree contents
+
+        @Feature: Content View
+
+        @Assert: CV should be created successfully with RH ostree contents
+
+        @Status: Manual
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier2
+    def test_positive_remove_custom_ostree(self):
+        """Create a CV with custom ostree contents and remove the
+        contents.
+
+        @Feature: Content View
+
+        @Assert: Content should be removed and CV should be updated
+        successfully
+
+        @Status: Manual
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier2
+    def test_positive_remove_rh_ostree(self):
+        """Create a CV with RH ostree contents and remove the
+        contents.
+
+        @Feature: Content View
+
+        @Assert: Content should be removed and CV should be updated
+        successfully
+
+        @Status: Manual
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier2
+    def test_positive_create_with_custom_ostree_other_contents(self):
+        """Create a CV with custom ostree contents and other custom yum, puppet
+        repos.
+
+        @Feature: Content View
+
+        @Assert: CV should be created successfully with all custom contents
+
+        @Status: Manual
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier2
+    def test_positive_create_with_rh_ostree_other_contents(self):
+        """Create a CV with RH ostree contents and other RH yum repos.
+
+        @Feature: Content View
+
+        @Assert: CV should be created successfully with all custom contents
+
+        @Status: Manual
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier2
+    def test_positive_publish_with_custom_ostree(self):
+        """Create a CV with custom ostree contents and publish it.
+
+        @Feature: Content View
+
+        @Assert: CV should be published with OStree contents
+
+        @Status: Manual
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier2
+    def test_positive_promote_with_custom_ostree(self):
```

Comment:
Can you please also add a test to undo a promotion? (remove the promoted content from an environment)